core/arch/arm/plat-bcm/conf.mk

```diff
@@ -16,14 +15,22 @@ CFG_TZDRAM_SIZE ?= 0x01000000 # 16MB
 CFG_SHMEM_START ?= ($(CFG_TZDRAM_START) - $(CFG_SHMEM_SIZE))
 CFG_SHMEM_SIZE ?=  0x01000000 # 16MB
 CFG_TEE_RAM_VA_SIZE := 0x400000 # 4MB
-endif
 
 ifeq ($(platform-flavor-armv8),1)
```

Comment:
minor: looks like `platform-flavor-armv8` is always 1.

Reply:
yes this condition is unnecessary now as all platforms are v8. And if we ever plan to add those v7 legacy platforms which is unlikely we can always have a separate plat-bcmv7 or similar.